app/views/appeal/discussion.scala

```diff
@@ -94,6 +97,10 @@ object discussion {
           )
         )
       ),
+      if (modData.isEmpty)
+      h2(
+        renderMark(user)
+      ),
```

Comment:
I doubt that works. You probably mean `modData.isEmpty option renderMark(user)`.

Reply:
Yeah. I could increase the font size by wrapping renderMark in h2, so it would be `modData.isEmpty option h2(renderMark(user))`